### DISCUSSION TITLE
Reading erasure shards at returned 'file not found', will attempt to reconstruct if we have quorum

### DISCUSSION THREAD

**Discussion by staybox** [minio/minio#18453] [Q&A] [answered]
Hello, we encountered this error.

Error: Reading erasure shards at (http://servername:9000/mnt/hdd/10: earth-services-v2/forest/order-files/552/input/633/data/polygons.dbf/979310a0-953c-4754-b691-32cb40794ae2/part.1) returned 'file not found', will attempt to reconstruct if we have quorum (*fmt.wrapError).

How to fix it?

**Comment by klauspost** [comment, marked as answer]
It will automatically heal. There is no need for any actions.

**Comment by julienlau** [comment]
It says "will attempt"
In the worst case scenario, could you please specify what would be the message if it cannot find the quorum to reconstruct the object ?

**Reply by klauspost** [reply]
The download will fail.

**Reply by julienlau** [reply]
Ok then the client will have a 4XX or 5XX.
But will it appear in the log of minio ?

**Reply by julienlau** [reply]
because 4xx and 5xx only appears in the trace to my knowledge
